### ISSUE TITLE
Solution explorer ability to move files

### ISSUE THREAD

**Issue by ekaradzha-qb** [microsoft/vscode-dotnettools#445]
### Move files with the solution explorer

One of the most missed features of the c# devkit solution explorer is the ability to move files to different folders. Also move to file/namespace refactoring action can complement this feature. VSCode default explorer does this at the moment, but it's very inconvenient to switch between the two views all the time.

### Alternatives considered

VSCode default file explorer

### Environment Information

OS: macOS
EXT VERSION: v0.4.6

**Comment by lukewis** [comment]
If you do move a file in the file explorer, all kinds of things break (red squigglies everywhere) and you have to close/re-open VS Code.  If you're moving more than one or two files, this is an immediate show-stopper.

**Comment by grimurd** [comment]
In a similar vein, copying files would also be a great addition.

**Comment by ekaradzha-qb** [comment]
Would be nice if it had, those complex operations: `Cut` `Copy` `Paste` `Delete` 😄

**Comment by julioct** [comment]
Yes please, having to switch between file view and solution explorer is very frustrating and confusing. 

In fact, let's get rid of solution explorer and have everything doable via the normal file explorer.

**Comment by timheuer** [comment]
@ekaradzha-qb drag/drop is in the latest version -- can you confirm you have latest and try it out?

**Comment by tomerbarkan** [comment]
> Yes please, having to switch between file view and solution explorer is very frustrating and confusing.
> 
> In fact, let's get rid of solution explorer and have everything doable via the normal file explorer.

The problem with the file explorer is that it's not grouped by projects, so copying or adding files there will not add them to the project. That's the main reason I'd like to see this functionality in the solution explorer, so that I can copy/drag files that will be added to the project without having to modify the .csproj files manually.
